__test__/pages/404/404.test.tsx

```diff
@@ -1,33 +1,24 @@
 import React from "react"
-import { render, screen } from "../../../src/utils/testUtils"
-
+import { render, screen, within } from "../../../src/utils/testUtils"
 import Custom404 from "../../../pages/404/index"
-import Redirect404 from "../../../pages/404/redirect"
 import { appConfig } from "../../../src/config/config"
 
 describe("404", () => {
   it("should display 404 text", () => {
-    render(<Custom404 />)
+    render(<Custom404 activePage="account" />)
+    const container = screen.getByRole("main")
 
-    const FourOhFourText = "404 Not Found"
-    const heading = screen.getByRole("heading")
-    expect(heading).toHaveTextContent(FourOhFourText)
+    const fourOhFourText = "404 Not Found"
```

Comment:
if you export this string from `pages/404/index.tsx` then it will be even less likely to break when updated later